### PR TITLE
Update end state to benchmark 1.2.0

### PR DESCRIPTION
**Pull request by keyboardsurfer**

### CODE REVIEW

File: benchmarking/macrobenchmark/src/main/java/com/example/macrobenchmark/ExampleStartupBenchmark.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 The Android Open Source Project
```
Comment (yenerm) [inline benchmarking/macrobenchmark/src/main/java/com/example/macrobenchmark/ExampleStartupBenchmark.kt:2]: 2023

File: benchmarking/macrobenchmark/src/main/java/com/example/macrobenchmark/ScrollBenchmarks.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 The Android Open Source Project
```
Comment (yenerm) [inline benchmarking/macrobenchmark/src/main/java/com/example/macrobenchmark/ScrollBenchmarks.kt:2]: 2023

File: baseline-profiles/baselineprofile/src/main/java/com/google/baselineprofile/StartupBenchmarks.kt [hunk 3]
```diff
@@ -0,0 +1,72 @@
+package com.google.baselineprofile
+
```
Comment (yenerm) [inline baseline-profiles/baselineprofile/src/main/java/com/google/baselineprofile/StartupBenchmarks.kt:2]: Add license

File: baseline-profiles/baselineprofile/src/main/java/com/google/baselineprofile/ScrollBenchmarks.kt [hunk 4]
```diff
@@ -0,0 +1,43 @@
+package com.google.baselineprofile
+
```
Comment (yenerm) [inline baseline-profiles/baselineprofile/src/main/java/com/google/baselineprofile/ScrollBenchmarks.kt:2]: missing license

File: baseline-profiles/baselineprofile/src/main/java/com/google/baselineprofile/BaselineProfileGenerator.kt [hunk 5]
```diff
@@ -0,0 +1,92 @@
+package com.google.baselineprofile
+
```
Comment (yenerm) [inline baseline-profiles/baselineprofile/src/main/java/com/google/baselineprofile/BaselineProfileGenerator.kt:2]: missing license

**Review by yenerm — CHANGES_REQUESTED** [review]

File: benchmarking/macrobenchmark/src/main/java/com/example/macrobenchmark/ExampleStartupBenchmark.kt (hunk 1 above)
Comment (keyboardsurfer) [reply benchmarking/macrobenchmark/src/main/java/com/example/macrobenchmark/ExampleStartupBenchmark.kt:2]: These were created in 2022. No need to update the copyright year.

**Review by yenerm — APPROVED** [review]

**Pull request merged**